data/devices/logitech-g705-wireless.device

```diff
@@ -0,0 +1,8 @@
+[Device]
+Name=Logitech G705
+DeviceMatch=usb:046d:c547
+Driver=hidpp20
+DeviceType=mouse
+
+[Driver/hidpp20]
+DeviceIndex=1
```

Comment:
> As as understand it, the kernel is supposed to support logitech receiver, so that each device connected to the receiver appears as a wired one, but that's not happening with this device.

You are right, and this file has deleted, sadly.

Reply:
I removed it and added the pid the device reports when the receiver is actually supported by the kernel.

I was able to get it to work by applying the patch from this issue to kernel module hid-logitech-dj
https://github.com/libratbag/libratbag/issues/1119 (Issue about the same receiver (pid 0xc547))

Apparently this patch was included at some point in kernel 6.6 but was reverted because it caused issues with other devices. (https://github.com/torvalds/linux/commit/5b4ffb176d7979ac66b349addf3f7de433335e00)